Config.psm1

```diff
@@ -21,57 +21,70 @@ Import-Module "$localResourcesDir\ini.psm1"
         @{"Name" = "disk_layout"; "DefaultValue" = "BIOS";
-          "Description" = ""},
+          "Description" = "This parameter can be set to either BIOS or UEFI."},
         @{"Name" = "product_key";
-          "Description" = ""},
+          "Description" = "The product key for the OS selected."},
         @{"Name" = "extra_features";
-          "Description" = ""},
+          "Description" = "Name of the extra features to enable on the generated image.
+                           The $ExtraFeatures need be present in the ISO file."},
         @{"Name" = "force"; "DefaultValue" = $false; "AsBoolean" = $true;
-          "Description" = ""},
+          "Description" = "It will force the image generation when $RunSysprep is $False or the selected $SwitchName
+                           is not an external one. Use this parameter with caution because it can easily generate
+                           unstable images."},
         @{"Name" = "install_maas_hooks"; "DefaultValue" = $false; "AsBoolean" = $true;
-          "Description" = ""},
+          "Description" = "If set to true, MaaSHooks will be installed."},
         @{"Name" = "zip_password";
-          "Description" = ""},
+          "Description" = "This parameter allows to create a password protected zip from the generated image."},
         @{"Name" = "administrator_password"; "GroupName" = "vm"; "DefaultValue" = "Pa`$`$w0rd";
-          "Description" = ""},
+          "Description" = "Is used by the script to auto-login in the instance while it is generating."},
         @{"Name" = "external_switch"; "GroupName" = "vm"; "DefaultValue" = "external";
-          "Description" = ""},
+          "Description" = "Used to specify the virtual switch the VM will be using to connect to the internet.
+                           If none is specified, one will be created."},
         @{"Name" = "cpu_count"; "GroupName" = "vm"; "DefaultValue" = "1";
-          "Description" = ""},
+          "Description" = "The number of CPU cores assigned to the VM used to generate the image."},
         @{"Name" = "ram_size"; "GroupName" = "vm"; "DefaultValue" = "2147483648";
-          "Description" = ""},
+          "Description" = "RAM assigned to the VM used to generate the image."},
         @{"Name" = "disk_size"; "GroupName" = "vm"; "DefaultValue" = "42949672960";
-          "Description" = ""},
+          "Description" = "Disk space assigned to the VM used to generate the image."},
         @{"Name" = "virtio_iso_path"; "GroupName" = "drivers";
-          "Description" = ""},
+          "Description" = "The path to the ISO file containing the VirtIO drivers."},
         @{"Name" = "virtio_base_path"; "GroupName" = "drivers";
-          "Description" = ""},
+          "Description" = "The drive letter of the mounted VirtIO drivers ISO file."},
         @{"Name" = "drivers_path"; "GroupName" = "drivers";
-          "Description" = ""},
+          "Description" = "The location of the drivers files."},
```

Comment:
"The location where additional drivers that are needed for the image are located."